Asterisk.2013/Asterisk.NET/FastAGI/AGIScript.cs

```diff
@@ -268,17 +268,20 @@ protected internal void SetPriority(string label)
 		{
 			this.Channel.SendCommand(new Command.SetPriorityCommand(label));
 		}
-		#endregion
-
-		#region StreamFile(string file)
-		/// <summary>
-		/// Plays the given file.
-		/// </summary>
-		/// <param name="file">name of the file to play.</param>
-		protected internal void StreamFile(string file)
-		{
-			this.Channel.SendCommand(new Command.StreamFileCommand(file));
-		}
+        #endregion
+
+        #region StreamFile(string file)
+        /// <summary>
+        /// Plays the given file.
+        /// </summary>
+        /// <param name="file">name of the file to play.</param>
+        /// <returns>failure: -1; failure on open: 0; success: 0; digit pressed: (digit) > 0.</returns>

```

Comment:
This line seems weird. Is it copied from documentation?
I don't know how this works, so I can't say it is wrong.

Reply:
I guess it is [Asterisk 14 AGICommand_stream file](https://wiki.asterisk.org/wiki/display/AST/Asterisk+14+AGICommand_stream+file).
Maybe add a link to that in the summary comment.